action.yml

```diff
@@ -54,6 +54,9 @@ outputs:
   internal-be-careful-output:
     description: "the column-formatted output from pip-audit, wrapped as base64"
     value: "${{ steps.pip-audit.outputs.output }}"
+  summary-raw:
+    description: "the raw summary output from pip-audit, passed to the Job Summaries page"
+    value: "${{ steps.pip-audit.outputs.summary-raw }}"
```

Comment:
Nitpick, but IMO we should call this `markdown-summary` or similar -- "raw" to me implies a machine-readable format like JSON.

Reply:
renamed to `markdown-summary` and added a nod to the GH UI language (`raw markdown`) in `README.md`